app/code/Magento/Catalog/Block/Rss/Category.php

```diff
@@ -17,6 +17,11 @@
 class Category extends \Magento\Framework\View\Element\AbstractBlock implements DataProviderInterface
 {
     /**
+     * @var \Magento\Framework\Config\View
+     */
+    protected $configView;
```

Comment:
Please change the property access type to private

Reply:
@sivaschenko  i did please check it.

Reply:
@sidolov 

Also, pay attention there are failed static and unit tests for this PR
Can you please tell me why it is happening? it will help to fix that error

Reply:
@sunilit42 please click the "Details" link next to failed build and open the Build reports to see details test result details

Reply:
Some of line showing  error even i did not touch that file
https://testing-service.magento-community.engineering/reports/magento/magento2/pull/23533/adc23fab-c90f-42d5-9a42-b90836a100fc/15775/Statics/allure-report-ce/index.html#suites/51d3ab4d5d77a0bc6a7ea344e9e7e6be/c0ee99e55056befd/

Reply:
@sidolov  why it is showing error even i did not touch that line no, please help me,

Reply:
@sunilit42 it is asking you to fix the static tests for files that you've modified

Reply:
@sivaschenko please check report and you will see an error on that line which I did not modified

Reply:
@sunilit42 I'm sorry, but you have to fix all fails, even if you have not modified those lines. That's just how our tests currently work.

Reply:
@sivaschenko  i have no issue to fix that issue, but i tried PSR2, PSR1, Magento2 Code standards which standard i need to use for that?

Reply:
@sunilit42 test results provide detailed info on what is expected. To open the tests results click the Details link next to failed build:
![image](https://user-images.githubusercontent.com/2028541/61134635-f05b5a80-a4b7-11e9-861a-4d48826b1c5e.png)
That will take you to the "Checks" tab of the pull request. Then click on the report link:
![image](https://user-images.githubusercontent.com/2028541/61134749-2d275180-a4b8-11e9-813b-e25662349526.png)
That will take you to Allure report. Finally, take a look at the failed test details by clicking on failed tests:
![image](https://user-images.githubusercontent.com/2028541/61134863-6f509300-a4b8-11e9-8223-a63f95404848.png)
![image](https://user-images.githubusercontent.com/2028541/61134946-9313d900-a4b8-11e9-85ba-1d47882e51a4.png)

Feel free to chat to me in slack if you need any help

Reply:
@sivaschenko  i fixed it, please check it.

Also tell me which Ruleset are you using ? is PSR-1,Magento2 or something else?

I tried to all ruleset and no one working so i did manually

Reply:
@sivaschenko any this missing ? Otherwise go ahead to approve it:)